desktop/lib/menu/help-menu.js

```diff
@@ -30,8 +31,14 @@ module.exports = function( mainWindow ) {
 		{
 			label: 'How can we help?',
 			click: function() {
-				mainWindow.show();
-				ipc.showHelp( mainWindow );
+				// on login page - user logged out
+				let parsedURL = url.parse( mainWindow.webContents.getURL() );
+				if ( parsedURL.pathname === '/login' ) {
```

Comment:
We have more routes now, with Sign Up in the app so this doesn't cover all cases. It would be better to check if the user is currently logged in, instead.